types/node/globals.d.ts

```diff
@@ -871,25 +872,12 @@ declare namespace NodeJS {
 
         resourceUsage(): ResourceUsage;
 
-        /**
-         * EventEmitter
-         *   1. beforeExit
-         *   2. disconnect
-         *   3. exit
-         *   4. message
-         *   5. rejectionHandled
-         *   6. uncaughtException
-         *   7. unhandledRejection
-         *   8. warning
-         *   9. message
-         *  10. <All OS Signals>
-         *  11. newListener/removeListener inherited from EventEmitter
-         */
```

Comment:
This probably shouldn’t be removed.

Reply:
why? What is the benefit of this?

Reply:
It's basically the same content as the actual EventEmitter definitions and has no value to the reader, we had it in a couple other places and it got removed over time.

I'd keep the `EventEmitter` line tho.

Reply:
added back `EventEmitter`.